test/integration-test/bpf/text_64_64_reloc.c

```diff
@@ -1,5 +1,5 @@
 // clang-format off
-#include <linux/bpf.h>
+#include <vmlinux.h>
```

Comment:
Should this be something like:
```c
#ifdef __linux__
#include <linux/bpf.h>
#else
#include <vmlinux.h>
#endif
```

You could even add our own `linux.h` file to DRY this up.
If possible it's probably better to let the preprocessor decide what to include than chosing `vmlinux` in all cases.

Reply:
But why? We don't actually use any Linux data structures, we need this include to provide defines relied on by the libbpf headers.

Reply:
Oh, you want the ones from libbpf? Those aren't self contained, see https://github.com/libbpf/libbpf/pull/41#issuecomment-1668143392.

There really aren't any viable options if we want things to be self-contained. We could write our own header, but then we own it for life. If this one disappears, we can do that then.